composer.json

```diff
@@ -26,18 +26,18 @@
         "phpspec/prophecy":         "^1.5",
         "phpspec/php-diff":         "^1.0.0",
         "sebastian/exporter":       "^1.0 || ^2.0 || ^3.0",
-        "symfony/console":          "^3.2",
-        "symfony/event-dispatcher": "^3.2",
-        "symfony/process":          "^3.2",
-        "symfony/finder":           "^3.2",
-        "symfony/yaml":             "^3.2",
+        "symfony/console":          "^3.2 || ^4.0@beta",
```

Comment:
The `@beta` suffix should be dropped: the CI build will test it in the `DEPENDENCIES=dev` matrix build.

Reply:
Correct. 

Could this PR me updated and merged? There are a ton of other projects that depends on this library that are waiting for this.